### ISSUE TITLE
Preserve a directory structure in multi-file upload using glob

### ISSUE THREAD

**Issue by danielc2013** [stelligent/crossing-go#9]
Background:
There is another ticket to allow for glob patterns. In this case, there should be a switch which determines if the directory is flattened, or preserved in the upload

Depends On: https://github.com/stelligent/crossing-go/issues/8

ToDo:
Add a flag to `crossing-go put` which will determine if the directory structure is preserved on multi-file upload. Something like `--preserve, -p`. The default behavior should also be defined. 
Example:
`crossing-go put -k alias/crossing-go-key -p -b s3://my-big-bucket/sub-folder ./**/*.py
